src/test/java/org/broadinstitute/hellbender/tools/spark/validation/CompareDuplicatesSparkIntegrationTest.java

```diff
@@ -78,22 +80,39 @@ public void differentBamTest(File firstBam, File secondBam, File firstBamDiffs,
     public void testOutputFile() throws Exception {
         // These files are not the same and should throw an exception
         final String resourceDir = getTestDataDir() + "/validation/";
-        try (final File gatk4Bam = new File(resourceDir, "tmp.gatk4.CEUTrio.HiSeq.WGS.b37.ch20.1m-1m1k.NA12878.bam"),
-             final File dupfreeBam = new File(resourceDir, "clean.CEUTrio.HiSeq.WGS.b37.ch20.1m-1m1k.NA12878.bam"),
-             final File validation1 = new File(resourceDir, "compareDuplicatesOutputTest.output1.bam"),
-             final File validation2 = new File(resourceDir, "compareDuplicatesOutputTest.output2.bam");) {
-
-            ArgumentsBuilder args = new ArgumentsBuilder();
-            args.add("--" + StandardArgumentDefinitions.INPUT_LONG_NAME);
-            args.add(gatk4Bam.getCanonicalPath());
-            args.add("--" + CompareDuplicatesSpark.INPUT_2_SHORT_NAME);
-            args.add(dupfreeBam.getCanonicalPath());
-            args.add("-O");
-            args.add(createTempFile("differentBamTest1", ".bam"));
-            args.add("-O2");
-            args.add(createTempFile("differentBamTest2", ".bam"));
-
-            this.runCommandLine(args.getArgsArray());
+        final File gatk4Bam = new File(resourceDir, "tmp.gatk4.CEUTrio.HiSeq.WGS.b37.ch20.1m-1m1k.NA12878.bam");
+        final File dupfreeBam = new File(resourceDir, "unDuplicateMarked.gatk4.CEUTrio.HiSeq.WGS.b37.ch20.1m-1m1k.NA12878.bam");
+        final File validation1 = new File(resourceDir, "compareDuplicatesOutputTest.output1.bam");
+        final File validation2 = new File(resourceDir, "compareDuplicatesOutputTest.output2.bam");
+        final File diffbam1 = createTempFile("differentBamTest1", ".bam");
+        final File diffbam2 = createTempFile("differentBamTest2", ".bam");
+
+        ArgumentsBuilder args = new ArgumentsBuilder();
```

Comment:
nitpick that you're free to disregard, but I always prefer the more handholding argument methods like args.addInput, etc.